lib/context.js

```diff
@@ -11,14 +11,14 @@ module.exports = Context;
 /**
  * Initialize a new `Context`.
  *
- * @api private
+ * @private
```

Comment:
Why would a module exported function be private?

Reply:
Uh, don't think that's how JSDoc access is _supposed_ to be used though. It's done from the aspect of the module, not user.

Reply:
Geez, you already merged this too?